protos/feast/core/Entity.proto

```diff
@@ -44,6 +44,9 @@ message EntitySpecV2 {
     // Description of the entity.
     string description = 3;
 
+    // Join key for the entity (i.e. name of the column it represents).
```

Comment:
Should we call this `default_join_key`, or is it impossible to override?

Reply:
currently you would use a feature map to map the column to this value rather than override the join key